src/aleph/handlers/content/vm.py

```diff
@@ -323,6 +323,9 @@ async def check_balance(self, session: DbSession, message: MessageDb) -> None:
             if not content.on.persistent:
                 return
 
+        if content.payment and content.payment.is_stream:
+            return
```

Comment:
how do you ensure that a payment stream exists?

Reply:
It is only checked on the CRNs and should check it also on the Scheduler, on the CCNs we just accept the message.